WORKSPACE

```diff
@@ -20,23 +20,40 @@ http_archive(
     url = "https://github.com/aspect-build/rules_js/archive/refs/tags/v1.8.0.tar.gz",
 )
 
-load("@aspect_rules_js//js:repositories.bzl", "rules_js_dependencies")
-
-rules_js_dependencies()
-
 http_archive(
     name = "rules_nodejs",
     sha256 = "50adf0b0ff6fc77d6909a790df02eefbbb3bc2b154ece3406361dda49607a7bd",
     urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/5.7.1/rules_nodejs-core-5.7.1.tar.gz"],
 )
 
+http_archive(
+    name = "aspect_rules_ts",
+    sha256 = "5b501313118b06093497b6429f124b973f99d1eb5a27a1cc372e5d6836360e9d",
+    strip_prefix = "rules_ts-1.0.2",
+    url = "https://github.com/aspect-build/rules_ts/archive/refs/tags/v1.0.2.tar.gz",
+)
+
+http_archive(
+    name = "aspect_rules_jest",
+    sha256 = "dd596891aa893048d2e8d210fce214459df33d454bf0e77906ebbfaee38f2bbc",
+    strip_prefix = "rules_jest-0.12.1",
+    url = "https://github.com/aspect-build/rules_jest/archive/refs/tags/v0.12.1.tar.gz",
+)
+
+# Node toolchain setup ==========================
 load("@rules_nodejs//nodejs:repositories.bzl", "DEFAULT_NODE_VERSION", "nodejs_register_toolchains")
 
 nodejs_register_toolchains(
     name = "nodejs",
     node_version = DEFAULT_NODE_VERSION,
 )
 
+# rules_js setup ================================
```

Comment:
Not a big fan of these headers 🤷‍♂️

Reply:
You don't like my beautiful art? :(

I just find the WORKSPACE becomes such a mess I like trying to separate things. Up to you though, I'll delete them?